pcsx2/Pcsx2Config.cpp

```diff
@@ -650,15 +653,23 @@ void Pcsx2Config::GSOptions::MaskUserHacks()
 #endif
 }
 
-void Pcsx2Config::GSOptions::MaskUpscalingHacks()
+void Pcsx2Config::GSOptions::UpscalingHacks()
 {
-	if (UpscaleMultiplier == 1 || ManualUserHacks)
```

Comment:
We discussed this on discord, but the only issue here is the condition should be inverted (`if (UpscaleMultiplier != 1 && !ManualUserHacks)`) in `MaskUpscalingHacks()`, no need to create an additional function.